src/main/java/frc/robot/Robot.java

```diff
@@ -32,23 +29,24 @@ public void robotInit() {
             Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
             Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
             new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
-        } else if (Constants.currentMode == Constants.Mode.SIM) {
```

Comment:
Why are we reverting this stuff? Shouldn't we maintain a sim mode separate from replay mode.

Also why are we doing this in this PR? I would expect renaming only

Reply:
I think this was done in Aiden's reformatting branch- this is branched off of that- and there was an issue with the replaying log?